### PR TITLE
Spaceworm

### PR DESCRIPTION
**Pull request by Breadbox2k19**
## Additions

Spaceworm

Check tests you have performed:
- [x ] Compiles without warnings
- [x ] Game runs without errors
- [x ] (For compatibility patches) ...with and without patched mod loaded
- [x ] Playtested a colony (specify how long)

### CODE REVIEW

File: Patches/Spaceworm/Spaceworm.xml [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+	<Operation Class="PatchOperationFindMod">
+		<mods>
+		<li>Space Worms (Continued)</li>
+		</mods>
+
+		<match Class="PatchOperationSequence">
+			<operations>
+
+			<li Class="PatchOperationAddModExtension">
+				<xpath>/Defs/ThingDef[defName="FaceTeeth"]</xpath>
```
Comment (N7Huntsman) [inline Patches/Spaceworm/Spaceworm.xml:12]: FaceTeeth is the defName of the BodyPartGroupDef. The defName for the creature is Scuttlebug.

File: Patches/Spaceworm/Spaceworm.xml [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+				<value>
+					<tools>
+						<li Class="CombatExtended.ToolCE">
+							<label>face teeth</label>
+							<capacities>
+								<li>ScuttlebugCapacity</li>
+							</capacities>
+							<power>5</power>
+							<cooldownTime>1</cooldownTime>
+							<linkedBodyPartsGroup>Teeth</linkedBodyPartsGroup>
```
Comment (N7Huntsman) [inline Patches/Spaceworm/Spaceworm.xml:40]: Duplicate linkedBodyPartsGroup tag here. Remove the Teeth one, keep the Mouth one.

**Review by N7Huntsman — CHANGES_REQUESTED** [review]

File: Patches/Spaceworm/Spaceworm.xml [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+							<armorPenetrationSharp>0.75</armorPenetrationSharp>
+							<armorPenetrationBlunt>0.844</armorPenetrationBlunt>
+        							<linkedBodyPartsGroup>Mouth</linkedBodyPartsGroup>
+						</li>
```
Comment (N7Huntsman) [inline Patches/Spaceworm/Spaceworm.xml:44]: I'd do away w/ the headattacktool entirely, honestly. Just add `ensureLinkedBodyPartsGroupAlwaysUsable` to the mouth attack.

**Review by N7Huntsman — CHANGES_REQUESTED** [review]

**Comment by Breadbox2k19** [comment]
All done.

**Comment by N7Huntsman** [comment]
> All done.

Sorry, to clarify what I said about removing the headattacktool, I would do away with the second blunt attack entirely, since the worms' gimmick is their bite attack that implants what are basically chest-bursters. When you do this, I'd add the `ensureLinkedBodyPartsGroupAlwaysUsable` to the mouth attack, just to ensure they can't be left w/o an attack on the off chance one gets their jaw destroyed.

**Review by N7Huntsman — APPROVED** [review]
Generally in good shape, I'll make a couple of quick revisions.

**Pull request merged**